### DISCUSSION TITLE
Is it possible to pass a trained model to the ProjectedGradientDescentPyTorch function?

### DISCUSSION THREAD

**Discussion by awskkzl** [Trusted-AI/adversarial-robustness-toolbox#1524] [Q&A]
So basically I have a trained model and I need to use PGD to generate adversarial samples for it. 
I notice that in the demo you build an ART classifier and fit the data in the ART classifier, then pass the classifier to PGD method.
Is it possible to pass a loaded model to PGD directly? I tried but I got "module has no 'all_framework_preprocessing'"
Please help

**Comment by beat-buesser** [comment]
Hi @awskkzl Yes, you need to provide the trained model first to `PyTorchClassifer`, for example similar to `examples/get_started_pytorch.py`.
